examples/01-produce-key-value/src/main.rs

```diff
@@ -23,17 +23,22 @@
 
 #[async_std::main]
 async fn main() {
+    let _ = tracing_subscriber::fmt()
+        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .try_init();
+
     if let Err(e) = produce_key_value().await {
         println!("Produce error: {:?}", e);
     }
 }
 
 async fn produce_key_value() -> Result<(), fluvio::FluvioError> {
-    let producer = fluvio::producer("key-value").await?;
+    let producer = fluvio::producer("twop").await?;
```

Comment:
`twop`?

Reply:
Whoops! I changed that for testing out two partitions. I'll switch it back